nodejs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "typedb-client",
-  "version": "2.11.1",
+  "version": "0.0.0",
```

Comment:
This version was outdated in the first place, and it gets overridden by `assemble_npm` anyway. Having any version other than a placeholder is misleading.